### PR TITLE
Update new users with infrequent use targeting profile age value

### PR DESCRIPTION
**Pull request by lmegviar**
See [original PR here](https://github.com/mozilla/experimenter/pull/8073)

This targeting has not yet been used and was newly added for an upcoming experiment. The original PR incorrectly used the `PROFILE28DAYS` variable. What we actually want is to target users with profile ages of _less_ than 28 days.

### CODE REVIEW

File: app/experimenter/targeting/constants.py [hunk 1]
```diff
@@ -464,8 +464,11 @@ def __post_init__(self):
 NEW_USERS_WITH_INFREQUENT_USE = NimbusTargetingConfig(
     name="New users with infrequent use",
     slug="new_users_with_infrequent_use",
-    description="0 - 6 days activity in past 28 days and profile age <= 28 days",
-    targeting=(f"{PROFILE28DAYS} " "&& userMonthlyActivity|length <= 6"),
+    description="0 - 6 days activity in past 28 days and profile age < 28 days",
+    targeting=(
+        "(currentDate|date - profileAgeCreated|date) / 86400000 < 28 "
```
Comment (yashika-khurana) [inline app/experimenter/targeting/constants.py:469]: I would suggest moving this calculation to a variable like `PROFILELESSTHAN28DAYS`

**Review by yashika-khurana — CHANGES_REQUESTED** [review]
Thank you for the clear description, I just have one suggestion, other than that it looks good!

**Comment by lmegviar** [comment]
Good call @yashika-khurana, I've moved that logic into a variable.

**Review by yashika-khurana — APPROVED** [review]
thank you @lmegviar looks perfect 💯

**Pull request merged**